alpine/3.10/7.3/Dockerfile

```diff
@@ -158,8 +159,6 @@ RUN addgroup spryker && \
 
 USER spryker
 ENV COMPOSER_MEMORY_LIMIT=-1
-RUN mkdir -p /home/spryker/.composer && \
```

Comment:
We still need this for composer v1. Can we have optional installation here?

Like,

bash -c '[[ $(composer --version) == "2*" ]] && install || true